PA.tex

```diff
@@ -0,0 +1,173 @@
+\section{Optimisation for Processor Architectures
+\label{sec:pa}}
+
+\subsection{NVIDIA GPU}
+The LFRic-microbenchmark suite~\cite{lfric-microbenchmarks} allows
+optimisations for individual kernels to be explored without having to
+compile and run the whole code. This is particularly useful when
+targeting a programming model such as OpenACC for GPU architecture
+where the Portland group compiler cannot compile the Fortran 2003 used 
+in the main code.
+
+The local matrix assembly (LMA) matrix-vector kernel had been
+previously ported to run on GPUs by inserting Open ACC directives in
```

Comment:
Here and throughout: "Open ACC" to "OpenACC" (no space)

Reply:
Yes, done.